wire-schema/src/jvmTest/kotlin/com/squareup/wire/schema/RootTest.kt

```diff
@@ -34,7 +34,7 @@ class RootTest {
 
     val location = Location.get("sample/src/main/proto", "squareup/dinosaurs/dinosaur.proto")
     val roots = location.roots(fs)
-    assertThat(roots.size == 1)
+    assertThat(roots.size == 1).isTrue()
```

Comment:
Can we do

```suggestion
    assertThat(roots.size).isEqualTo(1)
```

instead? The failure message will be much more descriptive (e.g. "expected <1> but was <5>" instead of "expected true but was false").

Reply:
Probably yes. This was an old test that I fixed because it was not testing *anything*